### PR TITLE
fix grammatical issues in CSS layout learning

### PR DESCRIPTION
**Pull request by yashikabhargava**
### Description

Fixed some grammatical issues in index.md.

### Motivation

To provide more clarity to the readers.

### CODE REVIEW

File: files/en-us/learn/css/css_layout/introduction/index.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ The methods that can change how elements are laid out in CSS are:
...
+The main methods for achieving page layout in CSS involve specifying values for the `display` property. This property allows us to change the default way something displays. Everything in normal flow has a default value for `display`; i.e., a default way that elements are set to behave. For example, the fact that paragraphs in English display one below the other is because they are styled with `display: block`. If you create a link around some text inside a paragraph, that link remains inline with the rest of the text, and doesn't break onto a new line. This is because the {{htmlelement("a")}} element is `display: inline` by default.
```
Comment (Josh-Cena) [inline files/en-us/learn/css/css_layout/introduction/index.md:96]: The original is not incorrect.

File: files/en-us/learn/css/css_layout/introduction/index.md [hunk 2]
```diff
@@ -107,7 +107,7 @@ Flexbox is the short name for the [Flexible Box Layout](/en-US/docs/Web/CSS/CSS_
...
+However, if we add `display: flex` to the parent, the three items now arrange themselves into columns. This is due to them becoming _flex items_ and being affected by some initial values that flexbox sets on the flex container. They are displayed in a row because the property {{cssxref("flex-direction")}} of the parent element has an initial value of `row`. They all appear to stretch in height because the property {{cssxref("align-items")}} of their parent element have an initial value of `stretch`. This means that the items stretch to the height of the flex container, which in this case is defined by the tallest item. The items all line up at the start of the container, leaving any extra space at the end of the row.
```
Comment (Josh-Cena) [inline files/en-us/learn/css/css_layout/introduction/index.md:110]: The subject for "has" is "the property `align-items`", which is singular.

File: files/en-us/learn/css/css_layout/introduction/index.md [hunk 3]
```diff
@@ -179,7 +179,7 @@ While flexbox is designed for one-dimensional layout, Grid Layout is designed fo
...
 
-Similar to flexbox, we enable Grid Layout with its specific display value — `display: grid`. The below example uses similar markup to the flex example, with a container and some child elements. In addition to using `display: grid`, we also define some row and column _tracks_ for the parent using the {{cssxref("grid-template-rows")}} and {{cssxref("grid-template-columns")}} properties respectively. We've defined three columns, each of `1fr`, as well as two rows of `100px`. We don't need to put any rules on the child elements; they're automatically placed into the cells our grid's created.
+Similar to flexbox, we enable Grid Layout with its specific display value — `display: grid`. The below example uses similar markup to the flex example, with a container and some child elements. In addition to using `display: grid`, we also define some row and column _tracks_ for the parent using the {{cssxref("grid-template-rows")}} and {{cssxref("grid-template-columns")}} properties respectively. We've defined three columns, each of `1fr`, as well as two rows of `100px`. We don't need to put any rules on the child elements; they're automatically placed into the cells our grids created.
```
Comment (Josh-Cena) [inline files/en-us/learn/css/css_layout/introduction/index.md:182]: The "'s" is a contraction for "has". This is a bit confusing, so let's get rid of the contraction instead.

**Review by Josh-Cena — CHANGES_REQUESTED** [review]

File: files/en-us/learn/css/css_layout/introduction/index.md (hunk 3 above)
Comment (yashikabhargava) [reply files/en-us/learn/css/css_layout/introduction/index.md]: Yes, I've done that, can you please review it again.

File: files/en-us/learn/css/css_layout/introduction/index.md [hunk 4]
```diff
@@ -563,15 +563,15 @@ body {
...
 
 The way that a table looks on a webpage when you use table markup is due to a set of CSS properties that define table layout. These same properties can also be used to lay out elements that aren't tables, a use which is sometimes described as "using CSS tables".
 
-The example below shows one such use. It must be noted, using CSS tables for layout should be considered a legacy method at this point, for those situations where you have very old browsers that lack support for Flexbox or Grid.
+The example below shows one such use. It must be noted, that using CSS tables for layout should be considered a legacy method at this point, for those situations where you have very old browsers that lack support for Flexbox or Grid.
```
Comment (Josh-Cena) [inline files/en-us/learn/css/css_layout/introduction/index.md:574]:
```suggestion
The example below shows one such use. It must be noted, that using CSS tables for layout should be considered a legacy method at this point, and should only be used to support old browsers that lack support for Flexbox or Grid.
```

**Review by Josh-Cena — APPROVED** [review]

**Pull request merged**